README.md

```diff
@@ -1 +1,76 @@
-# Challenges
\ No newline at end of file
+# Juice Shop Challenges

```

Comment:
nice one! You should also a section about why these attack types might be very dangerous, e.g. why is a possible sql injection bad? not only regarding login, but also for example for order-processes in e-commerce?

Reply:
thats is in Dangers, why sql injection is bad :D